lighthouse-core/lib/traces/tracing-processor.js

```diff
@@ -246,6 +235,28 @@ class TraceProcessor {
   }
 
   /**
+   * Provides the top level events on the main thread.
+   * @param {!traceviewer.Model} model
+   * @param {{traceEvents: !Array<!Object>}} trace
+   * @param {number=} startTime Optional start time (in ms) of range of interest. Defaults to trace start.
+   * @param {number=} endTime Optional end time (in ms) of range of interest. Defaults to trace end.
+   * @return {!Array<{start: number, end: number}>}
+   */
+  static getMainThreadTopLevelEvents(model, trace, startTime = 0, endTime = Infinity) {
```

Comment:
> hm, that actually feels a little less self-evident to me as someone who doesn't know the shape and meaning of the model properties well

`-Infinity` for `startTime`, then?

> we also wouldn't be able to specify them in the function signature anymore

that would actually work. You can refer to earlier params in default parameters